src/index.js

```diff
@@ -324,7 +326,8 @@ async function watch(stateManager, mainIndex) {
   });
 }
 
-function error(err) {
-  log.error(err);
+async function error(err) {
+  sentry.report(err);
+  await wait(5000);
   process.exit(1); // eslint-disable-line no-process-exit
```

Comment:
if we just rethrow the error, will sentry still catch it, or do we need the timeout?

Reply:
The timeout is needed (I will add a comment) if I don't wait the `process.exit` come too soon and the error don't have time to reach their API (I guess)